### PR TITLE
Hoc2022 - Code.org HoC Overview page updates

### PR DESCRIPTION
**Pull request by kelbyhawn**
Updates to https://code.org/hourofcode/overview for Hour of Code 2022:
- Rearrange activities to highlight Hello World and other activities relevant to the Explore, Play, Create theme
- Add a skinny banner that highlights and links to the new Explore, Play, Create landing page: https://code.org/hourofcode2022 — this is shown independently of hoc-mode and will be manually removed in Jan '23

**Relevant info:** [Asana task](https://app.asana.com/0/0/1202773411837028)

----

### Before:
code org_hourofcode_overview

### After:
localhost code org_3000_hourofcode_overview

----

**Skinny banner tablet:**
Screen Shot 2022-10-27 at 9 58 40 AM

**Skinny banner mobile:**
Screen Shot 2022-10-27 at 9 58 46 AM

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/hourofcode/overview.haml [hunk 1]
```diff
@@ -1,16 +1,12 @@
 ---
 title: Hour of Code Teacher Resources
 theme: responsive
-social:
```
Comment (bethanyaconnor) [inline pegasus/sites.v3/code.org/public/hourofcode/overview.haml:4]: Why remove these?

File: pegasus/sites.v3/code.org/public/hourofcode/overview.haml [hunk 2]
```diff
@@ -19,17 +15,24 @@ social:
 %div{dir: language_dir_class}
 
   %h1= I18n.t(:hoc_overview_title)
+  %aside.hoc-skinny-banner
+    .wrapper
+      %h1=hoc_s(:hoc2022_codeorg_title)
+      %p=hoc_s(:hoc2022_codeorg_description)
+    %img{src: "/images/hoc2022-skinny-banner-james-webb-illustration.png", alt: "Illustration of NASA's James Webb telescope"}
```
Comment (bethanyaconnor) [inline pegasus/sites.v3/code.org/public/hourofcode/overview.haml]: Maybe not something to handle in this PR but is it possible to get the alt text in the i18n pipeline?

**Review by bethanyaconnor — APPROVED** [review]

File: pegasus/sites.v3/code.org/public/hourofcode/overview.haml (hunk 1 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/hourofcode/overview.haml:4]: The opengraph tags for this page are being implemented on `social_metadata.rb` so I cleaned up the old tags. https://github.com/code-dot-org/code-dot-org/blob/193e93297dc30ce23772b2cf50b1fdccfe833db3/pegasus/src/social_metadata.rb#L144-L148

File: pegasus/sites.v3/code.org/public/hourofcode/overview.haml (hunk 2 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/hourofcode/overview.haml]: Yes definitely, good catch! I can add it right now. 

_Update:_ I added the string `hoc2022_nasa_webb_alt_text` to the i18n gsheet.

**Pull request merged**